src/juxt/site/alpha/graphql.clj

```diff
@@ -170,7 +170,7 @@
 
                    :else
                    (try
-                     (merge acc value)
+                     (assoc acc key value)
```

Comment:
this potentially will break something someone else was doing? but it works for me on tb + my drawing thing and I think its the correct thing to do. this code path is only done when the value is not a scalar, which means it must be a custom type, which means you want to assoc it on